apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

```diff
@@ -576,8 +576,18 @@ private static void verifyCodedExtension(String url, String code, String display
     assertTrue(compare.equalsDeep(ex));
   }
 
-  private static void verifyCoverageStatus() {
-    assertEquals("active", coverage.getStatus().toCode());
+  private static void verifyCoverageStatus(String status) {
+    if (status != null && !"".equals(status)) {
+      if ("active".equals(status)) {
+        assertEquals("active", coverage.getStatus().toCode());
+      } else if ("cancelled".equals(status)) {
+        assertEquals("cancelled", coverage.getStatus().toCode());
+      } else if ("draft".equals(status)) {
+        assertEquals("draft", coverage.getStatus().toCode());
+      } else if ("entered-in-error".equals(status)) {
+        assertEquals("entered-in-error", coverage.getStatus().toCode());
+      }
```

Comment:
can we just make this `assertEquals(status, coverage.getStatus().toCode())` ?

Reply:
I was not very clear, sorry. I was thinking the the entire method body can be replaced with just one assertEquals.